File: app/src/main/java/com/example/asthmaapplication/main/mainpage/subpages/learningpages/LearningFragmentActivity.java
package com.example.asthmaapplication.main.mainpage.subpages.learningpages;

import android.os.Bundle;

import com.example.asthmaapplication.R;
import com.example.asthmaapplication.main.common.BaseActivity;
import com.example.asthmaapplication.main.mainpage.subpages.LearningFragment;
import com.example.asthmaapplication.main.utils.ActivityUtils;

import dagger.hilt.android.AndroidEntryPoint;

@AndroidEntryPoint
public class LearningFragmentActivity extends BaseActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        LearningFragment fragment = new LearningFragment();
        ActivityUtils.addFragment(getSupportFragmentManager(), fragment, R.id.fragment_container);
    }
}

File: app/src/main/java/com/example/asthmaapplication/main/mainpage/subpages/learningpages/LearningFragmentActivity.kt
package com.example.asthmaapplication.main.mainpage.subpages.learningpages

import dagger.hilt.android.AndroidEntryPoint
import com.example.asthmaapplication.main.common.BaseActivity
import android.os.Bundle
import com.example.asthmaapplication.main.utils.ActivityUtils
import com.example.inventory.R

@AndroidEntryPoint
class LearningFragmentActivity : BaseActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        val fragment = LearningFragment()
        ActivityUtils.addFragment(supportFragmentManager, fragment, R.id.fragment_container)
    }
}